extensions/vscode/src/views/homeView.ts

```diff
@@ -405,15 +405,12 @@ export class HomeViewProvider implements WebviewViewProvider, Disposable {
     });
   }
 
-  private updateWebViewViewConfigurations(
-    deploymentSelector: DeploymentSelector | null,
-  ) {
+  private updateWebViewViewConfigurations() {
```

Comment:
No longer need to select the current configuration, since it is always derived out of the current contentRecord. Removing this also helped eliminate some race conditions I encountered.